### PR TITLE
fix type error in eq() call

### PR DESCRIPTION
**Pull request by zenogantner**
At least for my setup (torch 1.0.1, numpy 1.15.4) this is needed.

### CODE REVIEW

**Comment by TimDettmers** [comment]
This looks correct. I did not check the repo for PyTorch 1.0 yet — did you experience any other issues?

**Comment by zenogantner** [comment]
No other issues so far except for a deprecation warning:
UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.

**Comment by TimDettmers** [comment]
Sounds great, thank you!
